### DISCUSSION TITLE
Ability to play more than 3 sounds simultaneously

### DISCUSSION THREAD

**Discussion by oddMLan** [jenius-apps/ambie#205] [Ideas]
Many similar apps typically allow you to play more than 3 sounds simultaneously. Maybe it's kept low on purpose for simplicity's sake but if you want to make more elaborate soundscapes, it can be kind of limiting. My suggestion is to raise the limit to 5 sounds, or alternatively let the user customize the amount of sounds that can be played at the same time in the app settings.

**Comment by dpaulino** [comment]
The last time I tried having more than 3 active sounds, it wasn't a great UX. But I can try this again and see how the testing goes. If I think the UX is really bad, then I might not go with this. But if the UX is okay but not great, then I may provide it as an advanced option in settings. And of course, if the UX is great, then I can enable it as default. I'll let you know my findings when I get time to experiment with this.

**Reply by oddMLan** [reply]
@dpaulino Hi again. Thank you for the wonderful selection of sounds. Were you able to give this a spin?
I suppose the main thing going against this idea is ruining the beautiful simplicity of the UI.
This request is more of a feature parity with other soundscape apps that you can find on Android that allow up to 10 sounds play simultaneously, although this is probably overkill and would sound like a noisy mess.

~~https://github.com/jenius-apps/ambie/assets/7278372/3ce10956-0077-4de0-b7c2-aa4446eeb94d~~
~~I notice there's enough extra space on the left side to allow for an extra sound or two without looking cluttered. Thoughts?~~

Disregard that, I didn't have the latest version.
![image](https://github.com/jenius-apps/ambie/assets/7278372/a1464284-8cad-4062-a35c-3808c60f5fc7)

Since the newest update I see the sounds have been moved to the top. This definitely changes things. On a 16:9 monitor there's enough space to fit in two more sounds without overflow. 

![image](https://github.com/jenius-apps/ambie/assets/7278372/366e8d5e-5ce9-4a43-909e-750aad920e07)
Although splitting the window causes overflow of the third sound. The fourth sound wouldn't be visible. I see how this would cause UX issues.
